marketplace-and-integrations/integrations/algolia.md

```diff
@@ -54,6 +55,13 @@ The following configuration is required for working with the Algolia API:
 ```
 {% endcode %}
 
+Algolia comes with a set of predefined API keys:
+
+* Search-Only API key - a public API key used on the frontend for performing search queries.
+* Admin API key - used on the backend for create, update or delete operations on the indices. 
+
+More details on other use cases for the Algolia API keys can be found [here](https://www.algolia.com/doc/guides/security/api-keys/#predefined-api-keys).
```

Comment:
Using "here" for link texts, isn't great for SEO. Here's my suggestion for avoiding it:

```suggestion
More details on other use cases for the Algolia API keys can be found in [the Algolia Docs](https://www.algolia.com/doc/guides/security/api-keys/#predefined-api-keys).
```